app/Http/Controllers/Admin/AmaController.php

```diff
@@ -49,18 +50,40 @@ public function index(Request $request)
      */
     public function create()
     {
-        //
+        $ama_announcements = DB::table('ama_announcements')->select('title', 'user_id')->get();
+        return view('create', ['ama_announcements' => $ama_announcements]);
     }
 
     /**
-     * Store a newly created resource in storage.
-     *
      * @param  \Illuminate\Http\Request  $request
      * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+
+    public function save(Request $request)
     {
-        //
+
+        $this->validate($request, [
+            'title'   => 'required',
+            'content' => 'required',
+        ]);
+
+        $amas = new Ama();
+
+        $amas->title   = $request->input('title');
+        $amas->content = $request->input('content');
+
+        if ($request->tags) {
+            $amas->tags = $request->input('tags');
+        }
+        else{
+            $amas->tags = "";
+        }
+
+        $amas->person              = Auth::user()->name;
+        $amas->user_id             = Auth::user()->id;
+        $amas->ama_announcement_id = $request->input('ama_announcements');
+        $amas->save();
+        return redirect('/create')->with('success', true)->with('message', ' ');
```

Comment:
Set the message here, otherwise you won't know what to display. You don't need both, you can do `->with('success', 'Ama record successfully saved')` and then on frontend you'll read it with `flash('success')`